closed/src/java.base/share/classes/openj9/internal/criu/NativePRNG.java

```diff
@@ -215,9 +187,29 @@ private RandomIO(File seedFile, File nextFile) throws IOException {
             }
             this.seedIn = seedStream;
             this.nextIn = nextStream;
-            this.nextBuffer = new byte[bufferSize];
         }
 
+        // get the SHA1PRNG for mixing
+        // initialize if not yet created
+        private SHA1PRNG getMixRandom() {
+            SHA1PRNG r = mixRandom;
+            if (r == null) {
+                synchronized (LOCK_GET_SEED) {
+                    r = mixRandom;
+                    if (r == null) {
+                        try {
+                            byte[] b = new byte[20];
```

Comment:
I'm not a fan of one-letter variable names (i.e. `b` and `r`), surely we can be more descriptive.